sklearn/decomposition/_fastica.py

```diff
@@ -653,6 +665,7 @@ def inverse_transform(self, X, copy=True):
         Returns
         -------
         X_new : ndarray of shape (n_samples, n_features)
+            Transformed data obtained with the mixing matrix.
```

Comment:
The following makes it more explicit that this is about reconstructing signal in the original data space, I believe:

```suggestion
            Reconstructed data obtained with the mixing matrix.
```